python/gc_mlir/_mlir_libs/_site_initialize_0.py

```diff
@@ -20,4 +20,4 @@ def context_init_hook(context):
 
         register_onednn_graph_dialect(context)
     except ModuleNotFoundError:
-        print("onednn_graph dialect not found")
```

Comment:
Shall we still keep this error msg when trying with onednn_graph dialect instead of a silent pass?
If we don't want to use onednn graph dialect, then we can remove the `try...except...`?

Reply:
The `try ... except...` here is for the case when `GC_ENABLE_DNNL_API=OFF`, in that scenario, `onednn_graph` dialect is not included, but here the  `context_init_hook` is a hook that when using creating MLIR context in python, it will register these dialects automatically, the error msg will be a part of the python output which affect the integrity of your original python code.
I have fixed it now by getting the `GC_ENABLE_DNNL_API` value.
```
    if GC_ENABLE_DNNL_API:
        from ._gc_mlir.onednn_graph import (
            register_dialect as register_onednn_graph_dialect,
        )

        register_onednn_graph_dialect(context)
```